### DISCUSSION TITLE
Release v3.5.0

### DISCUSSION THREAD

**Discussion by NaBian** [HandyOrg/HandyControl#1522] [General]
## What's Changed
* 解决编译目标为x86的情况下报错 by @asine in https://github.com/HandyOrg/HandyControl/pull/1365
* [Bot] Automated PR to fix formatting errors by @github-actions in https://github.com/HandyOrg/HandyControl/pull/1337
* Update index.md by @kingsznhone in https://github.com/HandyOrg/HandyControl/pull/1380
* Update ISSUE_TEMPLATE from md to yaml by @JeremyWu917 in https://github.com/HandyOrg/HandyControl/pull/1266
* Add SECURITY.md by @JeremyWu917 in https://github.com/HandyOrg/HandyControl/pull/1281
* Czech language by @Zdendaki in https://github.com/HandyOrg/HandyControl/pull/1393
* [ImgBot] Optimize images by @imgbot in https://github.com/HandyOrg/HandyControl/pull/1472
* fix missing parentheses error in RegexPatterns.IpDPattern and RegexPatterns.IpEPattern by @infchaos in https://github.com/HandyOrg/HandyControl/pull/1225
* Add MenuItemAttach. by @GF-Huang in https://github.com/HandyOrg/HandyControl/pull/647

## New Contributors
* @asine made their first contribution in https://github.com/HandyOrg/HandyControl/pull/1365
* @kingsznhone made their first contribution in https://github.com/HandyOrg/HandyControl/pull/1380
* @Zdendaki made their first contribution in https://github.com/HandyOrg/HandyControl/pull/1393

**Full Changelog**: https://github.com/HandyOrg/HandyControl/compare/v3.4.0...v3.5.0

<hr /><em>This discussion was created from the release <a href='https://github.com/HandyOrg/HandyControl/releases/tag/v3.5.0'>Release v3.5.0</a>.</em>

**Comment by fARRRAA** [comment]
this is the best UI kit I've seen in my life, not that the best for wpf is one of the best UI kits on the Internet, even when compared with UI kits for web applications. TOP
